ingestion/src/metadata/ingestion/source/database/redshift/queries.py

```diff
@@ -22,7 +22,6 @@
     SELECT *
       FROM pg_catalog.stl_query
      WHERE userid > 1
-          {filters}
```

Comment:
if we remove the filter from here, could it increase the data fetched on the CTE? Now I'm not sure if we could push down the filters by adding them as soon as possible